model/project_vars.go

```diff
@@ -216,14 +223,41 @@ func (projectVars *ProjectVars) GetRestrictedVars() map[string]string {
 	return restrictedVars
 }
 
-func (projectVars *ProjectVars) GetAdminOnlyVars() map[string]string {
+// GetAdminOnlyVars will change to GetVars on EVG-16045 after removing restricted vars.
+func (projectVars *ProjectVars) GetAdminOnlyVars(t *task.Task) map[string]string {
```

Comment:
Can we also add a comment explaining who this returns variables for? (mainline commits + project admins)

Reply:
done